packages/client-search/src/types/HasPendingMappingsOptions.ts

```diff
@@ -2,14 +2,14 @@ export type HasPendingMappingsOptions = {
   /**
    * If the clusters pending mapping state should be on the response.
    *
-   * @default false
+   * @defaultValue false
```

Comment:
What is wrong with `@default`? I thought that was a valid ts-doc annotation.

Reply:
as far as I saw, @default is valid only in JSDoc, not in TSDoc + API Extractor as far as I could see: https://github.com/microsoft/tsdoc/issues/27

Reply:
https://api-extractor.com/pages/tsdoc/tag_defaultvalue/

Reply:
Thanks! ( I am sure you did ) make sure you update all `@default` to that new annotations.

Reply:
yes, those are the only two